lib/language_pack/rails51.rb

```diff
@@ -0,0 +1,85 @@
+require 'securerandom'
+require "language_pack"
+require "language_pack/rails5"
+
+class LanguagePack::Rails51 < LanguagePack::Rails5
+  NODE_MODULES_PATH = 'node_modules'
+  WEBPACKER_PACKS_PATH = 'public/packs'
+  WEBPACKER_CACHE_PATH = 'tmp/cache/webpacker'
+  YARN_CACHE_PATH = '~/.yarn-cache'
```

Comment:
FYI, yarn moved its cache location:
```
~ $ yarn cache dir
/app/.cache/yarn/v4
```

Reply:
Instead of a constant for each path, I've split the paths into two categories; ASSET_PATHS and ASSET_CACHE_PATHS, the latter being non-essential data that is removed after cleaning.